Resources/Prototypes/_Goobstation/Voice/speech_emotes.yml

```
@@ -121,3 +121,25 @@
     - ultra farted
     - ultra farting
+
+- type: emote 
+  id: Flap
+  name: chat-emote-name-flap
+  category: Vocal
+  available: false
+  icon: Interface/Emotes/chitter.png
+  whitelist:
+    components:
+      - Vocal
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)